qiskit/transpiler/passes/calibration/rx_builder.py

```diff
@@ -90,7 +90,7 @@ def __init__(
 
         super().__init__()
         self.target = target
-        self.already_generated = {}
+        self.already_generated = {}  # TODO: unused?
```

Comment:
Looks like this is unused

Reply:
indeed. Seems like it was introduced accidentally in a copy-paste in https://github.com/Qiskit/qiskit/pull/10634

Reply:
I'll just remove it then